care/users/tests/test_api.py

```diff
@@ -136,7 +137,7 @@ def test_user_can_read_all(self):
         self.assertEqual(response.status_code, status.HTTP_200_OK)
         res_data_json = response.json()
         # test total user count
-        self.assertEqual(res_data_json["count"], 2)  # 2 existing, plus the new one
+        self.assertEqual(res_data_json["count"], 3)  # 3 existing, plus the new one
```

Comment:
instead of hardcoding 3, we can fetch that value from the database. maybe in a separate ticket.